spec/classes/dns_init_spec.rb

```diff
@@ -151,6 +158,8 @@
         it { should contain_file(rndc_key) }
 
         it { should contain_service(service_name).with_ensure('running').with_enable(true).with_restart(nil) }
+
+        it { should contain_file("#{logdir}").with_ensure('directory') }
```

Comment:
No need to interpolate here
```suggestion
        it { should contain_file(logdir).with_ensure('directory') }
```